include/nccl_ofi_sendrecv.h

```diff
@@ -198,6 +200,9 @@ typedef struct nccl_net_ofi_sendrecv_req {
 	/* Associated Comm object */
 	nccl_net_ofi_comm_t *comm;
 
+	/* Associated inline buffer */
+	void *inline_buffer;
```

Comment:
Should we also wrap this in `#ifdef HAVE_NEURON`?

In general, some things related to inline writes are wrapped in HAVE_NEURON, and some are not, which is a bit confusing.